implementation/endpoints/include/udp_server_endpoint_impl_receive_op.hpp

```diff
@@ -306,11 +305,12 @@ receive_cb (std::shared_ptr<storage> _data) {
                     _data->sender_ = endpoint_type_t(its_sender_address, its_sender_port);
 
                     // destination
-                    struct in_pktinfo *its_pktinfo_v4;
                     for (struct cmsghdr *cmsg = CMSG_FIRSTHDR(&its_header);
                          cmsg != NULL;
                          cmsg = CMSG_NXTHDR(&its_header, cmsg)) {
 
+#if defined(IP_PKTINFO)
```

Comment:
nitpicking, but can you add a small comment to the if/elif/else with the OS types that should go in that part of the code.
It will make life easier to new devs working on this part of the code.

Reply:
Done